telemetry/specs/telemetry_report.mdk

```diff
@@ -473,49 +549,138 @@ particular deployment.
         [github.com/p4lang/switch](https://github.com/p4lang/switch/blob/master/p4src/includes/drop_reason_codes.h).
 
     See the INT specification [^INT] for definitions of the remaining metadata.
+    
+* Domain Specific ID (16b)
+  : The unique ID of the INT Domain
+
+* DSMdBits (16b): Domain Specific Md Bits
+  : Bitmap that indicates which optional domain specific metadata is 
+    present in the record of the telemetry report. Each bit represents 
+    4 octets or multiples of 4 octets of domain specific optional metadata. 
```

Comment:
When INT-MX sets a 'source-only' metadata bit, this introduces some complications for Telemetry Report 2.0 processing. At the source node, copying INT Hop ML into Telemetry Report MD Length will not work unless the resulting MD Length is increased to account for the 'source-only' metadata. At subsequent nodes, does that DSMdBit need to be cleared since that metadata will not be included in the telemetry report?

Reply:
I think the easiest thing to do is that the INT-MX source does not set the 'source-only' metadata bit in the data packet, but sets the bit and adjusts MD Length in the corresponding Telemetry Report 2.0 packet.

Reply:
I agree that source node shouldn't set the "Source Only" bit for INT-MX. Instead should set the "source only" bit in DSMdBits and adjusts the MD Length. I will add some text to capture this behavior.

Reply:
We should capture this INT-MX PR and here.

Reply:
+1 for restraining source node from using any source-only in INT-MX header but doing so in the Report 2.0 packet.